src/components/common/dashboard/Dashboard.vue

```diff
@@ -0,0 +1,324 @@
+                        (op) =>
+                            op.operationId.replace(/\s/g, "").toLowerCase().includes(filter) ||
+                            op.initiator.replace(/\s/g, "").toLowerCase().includes(filter) ||
+                            op.transactionInfo.parameters.toString().replace(/\s/g, "").toLowerCase().includes(filter)
+                    );
+                    return filteredOperations;
+                }
+                return ops;
+            }
+            const filteredOperations = computed(() => {
+                let filter = message.value.replace(/\s/g, "").toLowerCase();
+                if (filter != "") {
+                    //TODO more filtering
+                    let filteredOperations = operations.value.filter(
+                        (op) =>
+                            op.operationId.replace(/\s/g, "").toLowerCase().includes(filter) ||
+                            op.initiator.replace(/\s/g, "").toLowerCase().includes(filter) ||
+                            op.transactionInfo.parameters.toString().replace(/\s/g, "").toLowerCase().includes(filter)
+                    );
+                    return filteredOperations;
+                }
+                return operations.value;
+            });
+
+            const pendingOwnOperations = computed(() => {
+                let filter = message.value.replace(/\s/g, "").toLowerCase();
+                return filterOperations(operations.value, filter).filter(
+                    (op) => op.state == OperationStatus.PENDING && op.initiator == enrollmentId.value
+                );
+            });
+            const finishedOperations = computed(() => {
+                let filter = message.value.replace(/\s/g, "").toLowerCase();
+                return filterOperations(operations.value, filter).filter(
+                    (op) => op.state == OperationStatus.REJECTED || op.state == OperationStatus.FINISHED
+                );
+            });
+            const shownActionRequiredOperations = computed(() => {
+                let filter = message.value.replace(/\s/g, "").toLowerCase();
+                return filterOperations(actionRequiredOperations.value, filter).filter(
+                    (op) =>
+                        op.state == OperationStatus.PENDING &&
+                        (op.missingApprovals.users.includes(enrollmentId.value) || op.missingApprovals.groups.includes(role.value))
+                );
+            });
+
+            async function getOperations() {
+                busy.value++;
+                const operationManagement = new OperationManagement();
+                const handler = new GenericResponseHandler("operations");
+                let response = await operationManagement.getOperations(undefined, undefined, undefined, true);
+                let result = handler.handleResponse(response);
+                if (Object.keys(result).length > 0) {
+                    operations.value = result;
+                }
```

Comment:
operations should be renamed to watchlistOperations, as only those are returned here

Reply:
Changed in https://github.com/upb-uc4/ui-web/pull/785/commits/2c5f925f9401503b8bb5308e25bfca9a734f0544